tests/integration/components/bs-modal/header-test.js

```diff
@@ -16,7 +16,7 @@ module('Integration | Component | bs-modal/header', function(hooks) {
   });
 
   test('Header can have custom block content', async function(assert) {
-    await render(hbs`{{#bs-modal/header}}<div id="custom">Test</div>{{/bs-modal/header}}`);
+    await render(hbs`{{#bs-modal/header as |header|}}<div id="custom">Test</div>{{header.close}}{{/bs-modal/header}}`);
```

Comment:
When following the approach I suggested above, the existing test should remain as-is, and a new test for the yielded components case should be added.